src/main.js

```diff
@@ -33,10 +33,10 @@ function initAlpine() {
 		},
 		isMobile: true,
 	});
-	Alpine.store("currentDataset", {
-		geoJSON: null,
-		slug: null,
-	});
+	// Alpine.store("currentDataset", {
```

Comment:
No need to leave this commented out, we can always fetch the code from a previous commit.